src/components/book.js

```diff
@@ -6,23 +6,24 @@ import { removeBook } from '../redux/books/booksSlice';
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   return (
-    <>
+    <div className="Lesson-Panel">
       <p key={book.id}>
-        <strong>{book.title}</strong>
+        <strong className="Title">{book.title}</strong>
+        <br />
         {' '}
-        <span>by</span>
         {' '}
-        <strong>{book.author}</strong>
+        <strong className="author">{book.author}</strong>
       </p>
       <button
         type="button"
         onClick={() => {
           dispatch(removeBook(book.id));
         }}
+        className="Remove"
       >
-        Delete
+        Remove
       </button>
-    </>
+    </div>
```

Comment:
- [ ] You're missing out on other placeholder elements that are part of the design requirements of this project. Kindly match your design with the original design given. The book cards of your design are empty right now. Please add the other elements as required. 👍🏽
- [ ] Also, it would be nice if you could center the frame of the app just as the original design. Your main app is aligned to the left right now and it's not centered. Kindly work on that 👍🏽

**The Original Design**

<img width="1265" alt="Screenshot 2023-04-07 at 5 34 23 PM" src="https://user-images.githubusercontent.com/50941074/230653017-eaa54a59-ef5a-4112-bc4d-a4b23996028f.png">

**Your Design**

<img width="1776" alt="Screenshot 2023-04-07 at 5 31 43 PM" src="https://user-images.githubusercontent.com/50941074/230653058-27554835-62b3-4250-b12a-d5bbff4a0ea0.png">